connect_sample/src/main/java/com/zendesk/sample/connect/SampleNotificationService.java

```diff
@@ -0,0 +1,29 @@
+package com.zendesk.sample.connect;
+
+import android.content.Intent;
+import android.net.Uri;
+
+import com.zendesk.logger.Logger;
+
+import io.outbound.sdk.OutboundService;
+import io.outbound.sdk.PushNotification;
+
+/**
+ * You can optionally extend {@link OutboundService} to handle what happens when a user
```

Comment:
Similar to the above, can you expand on the optionality of this? It doesn't look like this class is referenced anywhere outside of this file - does it need to be?